keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -415,9 +438,13 @@ message PodSandboxStats {
     LinuxPodSandboxStats linux = 2;
     // Stats from windows.
     WindowsPodSandboxStats windows = 3;
+    // Generic Stats that apply across OS
+    PodSandboxStats = 4;
 }
 
 // LinuxPodSandboxStats provides the resource usage statistics for a pod sandbox on linux.
+// This field is here for backwards compatibility and future Linux only stats.
```

Comment:
I think it would be worth saying this field is explicity deprecated and may be removed in the future.